src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassDataAbstractionCouplingCheck.java

```diff
@@ -234,30 +337,22 @@
  * import a.b.Bar;
  * import a.b.c.Baz;
  *
- * public class Foo {
- *   public Bar bar; // Will be ignored, located inside ignored a.b package
- *   public Baz baz; // Will not be ignored, located inside a.b.c package
- *   public Data data; // Will not be ignored, same file
+ * class Foo {
+ *   Bar bar; // Will be ignored, located inside ignored a.b package
+ *   Baz baz; // Will not be ignored, located inside a.b.c package
+ *   Data data; // Will not be ignored, same file
```

Comment:
What is the reason of this refactoring? Please revert if not needed.

Reply:
https://github.com/checkstyle/checkstyle/pull/7906#discussion_r399785305
Please clarify.

Reply:
Ok, thanks for clarifying